src/main/java/com/rocketden/main/dto/room/JoinRoomRequest.java

```diff
@@ -0,0 +1,11 @@
+package com.rocketden.main.dto.room;
+
+import lombok.Getter;
+import lombok.Setter;
```

Comment:
Not a critique but I didn't know you could make getters and setters this way! note to self to try this in the future